tests/integration/cli/test_cli.py

```diff
@@ -12,49 +12,65 @@
 
 # This assumes have OTX installed in environment.
 otx_module = importlib.import_module("otx")
-RECIPE_PATH = Path(inspect.getfile(otx_module)).parent / "recipe"
-ALL_RECIPE_LIST = [str(_.relative_to(RECIPE_PATH)) for _ in RECIPE_PATH.glob("**/*.yaml")]
-RECIPE_OV_LIST = [str(_.relative_to(RECIPE_PATH)) for _ in RECIPE_PATH.glob("**/openvino_model.yaml")]
-RECIPE_LIST = set(ALL_RECIPE_LIST) - set(RECIPE_OV_LIST)
+RECIPE_PATH = Path(inspect.getfile(otx_module)).parent / "configs"
```

Comment:
Maybe not the scope of this PR but maybe we can use fixtures rather than global variables to get these values